bika/health/upgrade/v01_01_001.py

```diff
@@ -27,7 +28,9 @@ def upgrade(tool):
 
     logger.info("Upgrading {0}: {1} -> {2}".format(product, ver_from, version))
 
-    # -------- ADD YOUR STUFF HERE --------
+    # Required to make filtering by department in worksheets work
+    ut.addIndex(CATALOG_PATIENT_LISTING, 'getClientPatientID', 'FieldIndex')
+    ut.refreshCatalogs()
```

Comment:
Always remember to add the index (or column if necessary) in the catalog definition (https://github.com/senaite/senaite.health/blob/master/bika/health/catalog/patient_catalog.py#L29), otherwise, this index will only be available in instances that ran this upgrade step, but not in instances installed from scratch.